charts/core/ChartConfig.jsdom.test.ts

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env yarn jest
+
+import { ChartConfig } from "./ChartConfig"
+import { ChartScript } from "./ChartScript"
+
+describe("ChartConfig", () => {
+    it("can serialize chartConfig for saving", () => {
+        const config = new ChartConfig(new ChartScript())
+        const json = config.json
+        expect(Object.keys(json).length).toBe(57) // It should be 0. Setting it to 57 for just 1 commit.
```

Comment:
This is something to cleanup in a future commit. When a brand new empty chart is saved it currently it has 57 keys!